book/src/fundamentals/zkbook_commitment.md

```diff
@@ -36,7 +36,7 @@ If the hash function is one-way, then this commitment is also hiding.
 ## Algebraic and homomorphic commitments
 
 Instead of a cryptographic hash function, we can use elliptic curve cryptography to construct a commitment scheme.  Here elliptic curve scalar
-multiplication is used as the one way function.  Suppose we have
+multiplication is used like a one way function based on some hardness assumptions.  Suppose we have
```

Comment:
maybe based on the hardness assumption of the ECDLOG?

Reply:
yes exactly

Reply:
Nice -- I'll move this specific up from the section below.